File: chroma-league/src/main/java/com/bonepl/chromaleague/hud/animations/EnemyElderDragonKillAnimation.java
package com.bonepl.chromaleague.hud.animations;

import com.bonepl.chromaleague.rest.eventdata.DragonType;

public class EnemyElderDragonKillAnimation extends StaticPartialBlinkingAnimation {
    public EnemyElderDragonKillAnimation() {
        super(DragonType.ELDER.getColor());
    }
}

File: src/hud/animations/EnemyElderDragonKillAnimation.ts
import {DragonType, dragonTypeGetColor} from '../../rest/eventdata/DragonType.js';
import {StaticPartialBlinkingAnimation} from './StaticPartialBlinkingAnimation.js';

export class EnemyElderDragonKillAnimation extends StaticPartialBlinkingAnimation {
    constructor() {
        super(dragonTypeGetColor(DragonType.ELDER));
    }
}
